tasks/bridge_configuration.yml

```diff
@@ -1,6 +1,6 @@
 ---
-
 - name: Check active bridge interface state
+  become: true
```

Comment:
I don't think it's necessary for this task.

Reply:
you are right. i fixed it!